### ISSUE TITLE
Getting Unsupported startup parameter: statement_timeout error when connecting to PostgreSQL

### ISSUE THREAD

**Issue by vladimir-kotikov** [mtxr/vscode-sqltools#272]
**Describe the bug**
I'm getting the error when connecting to a remote Potgres:

```
INFO: Executing SQLTools.selectConnection
ERROR: Error:  { [Error: Unsupported startup parameter: statement_timeout
	at handleResponse (/Users/kotikov.vladimir/.vscode/extensions/mtxr.sqltools-0.19.5/extension.js:919:48)
	at processMessageQueue (/Users/kotikov.vladimir/.vscode/extensions/mtxr.sqltools-0.19.5/extension.js:746:17)
	at Immediate.module.exports.setImmediate (/Users/kotikov.vladimir/.vscode/extensions/mtxr.sqltools-0.19.5/extension.js:730:13)
	at runCallback (timers.js:696:18)
	at tryOnImmediate (timers.js:667:5)
	at processImmediate (timers.js:649:5)]
  code: -32001,
  data: { dialect: 'PostgreSQL', dialectOptions: {} } }
```

**To Reproduce**
Nothing special, just create a connection to local postgres and try to connect 

**Expected behavior**
It should connect flawlessly

**Screenshots**
![image](https://user-images.githubusercontent.com/3857604/59447590-2a2f3780-8e0c-11e9-9a6a-0d56ec4ed3fe.png)

**Desktop (please complete the following information):**
 - SQLTools Version latest available, 0.19.5
 - VSCode Version: latest, 1.35.1
 - Dialect Used: PostgreSQL
 - OS: Mac

**Additional context**
An unusual thing is that I connect to a remote Postgres which is tunneled to localhost through SSH but honestly I don't think it really makes any difference

**Comment by mtxr** [comment]
Hi, 

sorry for the late reply, I had to deal with some stuff here in the past week, I'll take a look at this within this week.

Thanks!

**Comment by mtxr** [comment]
can you share you settings? You can mask private stuff.

This is only used here, and it only breaks if connectionTimeout is a string.
```
statement_timeout: this.credentials.connectionTimeout * 1000,
```

**Comment by vladimir-kotikov** [comment]
I think it's very simple, no extra stuff. That's what I have in my workspace config file:
```json
"sqltools.connections": [
	{
		"connectString": "postgresql://user:password@localhost:6543/database",
		"connectionTimeout": 30,
		"dialect": "PostgreSQL",
		"name": "Vagrant"
	}
],
```

**Comment by vladimir-kotikov** [comment]
Alternative configuration produces the same error: 

```json
{
	"connectionTimeout": 30,
	"database": "database",
	"dialect": "PostgreSQL",
	"name": "Vagrant",
	"password": "password",
	"port": 6543,
	"server": "127.0.0.1",
	"username": "user"
}
```

Note that corresponding terminal command connects to DB just fine:

    psql -d database -h 127.0.0.1  -p 6543 -U user -W

**Comment by mtxr** [comment]
I've published a new version. Will be available to update in a few minutes. Can you test again with this new version?

It's weird to see this message for you setup. Looks just fine..

**Comment by vladimir-kotikov** [comment]
Updated and still seeng the same error :( Logs are even less verbose than last time and don't contain any errors

**Comment by mtxr** [comment]
@vladimir-kotikov  Apparently some PostgreSQL version doesn't support this as a startup parameter.

Which version are you using? I'm planning to switch the strategy for timeouts if need, but I want to confirm this hypothesis first

**Comment by mtxr** [comment]
Closing due lack of response.

**Comment by vladimir-kotikov** [comment]
Hey @mtxr, sorry for the delay. I'm using PostgreSQL version 10.9

```
$ psql 
psql (10.9 (Ubuntu 10.9-0ubuntu0.18.04.1))
Type "help" for help.

postgres=> SELECT version();
                                                               version                                                                
--------------------------------------------------------------------------------------------------------------------------------------
 PostgreSQL 10.9 (Ubuntu 10.9-0ubuntu0.18.04.1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0, 64-bit
(1 row)
 ```

however the issue does seem to be resolved now (extension version is 0.19.7) - I can connect to the same database successfully.
